handlers/server_test.go

```diff
@@ -0,0 +1,615 @@
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
+			}
+
+			w := httptest.NewRecorder()
+
+			//some fake uuid
+			vars := map[string]string{"id": "663c9932-383d-44d1-8b9d-1071731a6312"}
+
+			req = mux.SetURLVars(req, vars)
+
+			newEmployee := store.Employee{
+				Name:       "pablo",
+				Email:      "pablo@yahoo.com",
+				EmployeeId: "663c9932-383d-44d1-8b9d-1071731a6312",
+				City:       "Denver",
+				Address:    "3 Main St",
+				Department: "sales",
+			}
+
+			item, _ := dynamodbattribute.MarshalMap(newEmployee)
+
+			m.GetError = nil
+			m.GetOutput = dynamodb.GetItemOutput{
+				Item: item,
+			}
+
+			m.UpdateError = nil
+
+			server.UpdateEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp GetOneEmployee
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+			if err != nil {
+				fmt.Printf("unable to successfully decode response got error %v", err)
+			}
+
+			//refactor
+			Expect(resp.Status).To(Equal(http.StatusOK))
+			//update field for test
+			newEmployee.Department = "eng"
+			Expect(resp.Employee).To(Equal(newEmployee))
+
+		})
+	})
+
+	Context("Sad Path tests", func() {
+		It("Should test GET Employee (400), Invalid UUID", func() {
+
+			req, err := http.NewRequest("GET", "/v1/employee/{id}", nil)
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
+			}
+
+			w := httptest.NewRecorder()
+
+			//some invalid UUID
+			vars := map[string]string{"id": "ZZ3c9932-383d-44d1-8b9d-1071731a6312"}
+
+			req = mux.SetURLVars(req, vars)
+
+			server.GetEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp store.Error
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+
+			if err != nil {
+				fmt.Printf("unable to successfully decode response got error %v", err)
+			}
+
+			Expect(resp.Status).To(Equal(http.StatusBadRequest))
+			Expect(resp.Msg).To(Equal("GET: Not a valid uuid!"))
+		})
+		It("Should test GET Employee (404), getItem method error for not found user", func() {
+
+			req, err := http.NewRequest("GET", "/v1/employee/{id}", nil)
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
+			}
+
+			w := httptest.NewRecorder()
+
+			//some fake UUID
+			vars := map[string]string{"id": "663c9932-383d-44d1-8b9d-1071731a6312"}
+
+			req = mux.SetURLVars(req, vars)
+
+			m.GetError = nil
+			m.GetOutput = dynamodb.GetItemOutput{}
+
+			server.GetEmployee(w, req)
+
+			res := w.Result()
+			defer res.Body.Close()
+
+			var resp store.Error
+
+			err = json.NewDecoder(res.Body).Decode(&resp)
+
+			Expect(resp.Status).To(Equal(http.StatusNotFound))
+			Expect(resp.Msg).To(Equal("GET: Unable to find employee with that UUID!"))
+		})
+		It("Should test GET Employee (500), getItem method error in mock DynamoDB client", func() {
+
+			req, err := http.NewRequest("GET", "/v1/employee/{id}", nil)
+			if err != nil {
+				fmt.Printf("could not create a request: %v", err.Error())
```

Comment:
I see that you print errors to the console in a lot of places. If you assert that they're nil With Expect() calls, do the tests pass? This is a change I'd like you to make throughout all your unit tests to make sure your tests fail in these scenarios rather than just printing and continuing.